### ISSUE TITLE
update rscloud package to point to api.posit.cloud (instead of api.rstudio.cloud)

### ISSUE THREAD

**Issue by m--** [rstudio/rscloud#55]

**Comment by mine-cetinkaya-rundel** [comment]
I wonder if the package name should also be updated. Any thoughts? We haven't submitted to CRAN yet so change should be feasible.

cc @ssinnott

**Comment by ssinnott** [comment]
> I wonder if the package name should also be updated. Any thoughts? We haven't submitted to CRAN yet so change should be feasible.
> 
> cc @ssinnott

Yup! I think we should probably update the name of the package as well!
